internal/backend/crypto/age/askpass.go

```diff
@@ -38,7 +39,12 @@ func newAskPass() *askPass {
 	return &askPass{
 		cache: cache.NewInMemTTL(time.Hour, 24*time.Hour),
 		pinentry: func() (piner, error) {
-			return pinentry.New()
+			p, err := pinentry.New()
```

Comment:
Shouldn't this be provided by the cli itself through a function "TryPinEntry" or so?
It would render the cli package a bit more useful.

Reply:
That would require more changes than I'd like to do for now (introducing some interfaces and such).